.github/scripts/install-arduino-ide.sh

```diff
@@ -107,6 +108,7 @@ function build_sketch(){ # build_sketch <fqbn> <path-to-ino> [extra-options]
 		-libraries "$ARDUINO_USR_PATH/libraries" \
 		-build-cache "$ARDUINO_CACHE_DIR" \
 		-build-path "$ARDUINO_BUILD_DIR" \
+		-prefs=compiler.cpp.extra_flags="$build_flags" \
```

Comment:
this should maybe include the c flags as well, not just cpp

Reply:
Currently I'm only using a CPPDEFINE